Action Extension/ActionExtensionViewController.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2022 Metabolist. All rights reserved.
+
+import MobileCoreServices
+import UIKit
+import UniformTypeIdentifiers
+
+/// Let the user open a URL that might be a Mastodon profile or post in Metatext.
+/// Should activate for one or more web URLs, but will only do anything with the first..
+class ActionExtensionViewController: UIViewController {
+    /// Extensions aren't allowed to call `UIApplication.shared
```

Comment:
suggestion: move the /// comment from lines 7,8 to be the comment for func(open: and then move the existing comment from lines 10-15 inside the open function as an explanation for how it works, and update to // comments instead